src/SDK/Trace/TraceState.php

```diff
@@ -146,7 +146,7 @@ private function parse(string $rawTracestate): array
             }
 
             foreach ($listMembers as $listMember) {
-                $vendor = explode(self::LIST_MEMBER_KEY_VALUE_SPLITTER, $listMember);
+                $vendor = explode(self::LIST_MEMBER_KEY_VALUE_SPLITTER, trim($listMember));
```

Comment:
This fixes the tracestate test case that has spaces within `BaggagePropatagorTest`.